springwolf-core/src/main/java/io/github/stavshamir/springwolf/SpringWolfConfigProperties.java

```diff
@@ -0,0 +1,83 @@
+package io.github.stavshamir.springwolf;
+
+import com.asyncapi.v2.model.info.Contact;
+import com.asyncapi.v2.model.info.License;
+import com.asyncapi.v2.model.server.Server;
+import io.github.stavshamir.springwolf.configuration.AsyncApiDocket;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Map;
+
+@Configuration
+@ConfigurationProperties(prefix = SpringWolfConfigConstants.SPRINGWOLF_CONFIG_PREFIX)
+@ConditionalOnProperty(name = SpringWolfConfigConstants.SPRINGWOLF_ENABLED, matchIfMissing = true)
+@Getter
+@Setter
+public class SpringWolfConfigProperties {
+
+    private boolean enabled = true;
```

Comment:
What would happen if this is false?

Reply:
It will now disabled spring wolf completely.